raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -129,16 +133,41 @@ public String toString() {
                 QUORUM_LINGER_MS_DOC);
     }
 
+    private final int requestTimeoutMs;
+    private final int retryBackoffMs;
+    private final int electionTimeoutMs;
+    private final int electionBackoffMaxMs;
+    private final int fetchTimeoutMs;
+    private final int appendLingerMs;
+
     public RaftConfig(Properties props) {
         super(CONFIG, props);
+        requestTimeoutMs = getInt(QUORUM_REQUEST_TIMEOUT_MS_CONFIG);
+        retryBackoffMs = getInt(QUORUM_RETRY_BACKOFF_MS_CONFIG);
+        electionTimeoutMs = getInt(QUORUM_ELECTION_TIMEOUT_MS_CONFIG);
+        electionBackoffMaxMs = getInt(QUORUM_ELECTION_BACKOFF_MAX_MS_CONFIG);
+        fetchTimeoutMs = getInt(QUORUM_FETCH_TIMEOUT_MS_CONFIG);
+        appendLingerMs = getInt(QUORUM_LINGER_MS_CONFIG);
     }
 
-    public RaftConfig(Map<String, Object> props) {
+    public RaftConfig(Map<?, ?> props) {
         super(CONFIG, props);
+        requestTimeoutMs = getInt(QUORUM_REQUEST_TIMEOUT_MS_CONFIG);
```

Comment:
nit: the typical thing to do is invoke the other constructor. e.g. `this(props, true)`

Reply:
Ah thanks for catching that. Turns out both public constructors can use the protected constructor (Properties implements Map). Will refactor both.

Reply:
Actually, that means I can remove the `public RaftConfig(Properties props)` constructor. Does that make sense @hachikuji?

Reply:
Yeah, that's why I said it was redundant in the other comment 🙂

Reply:
Perfect. Removed.